### ISSUE TITLE
Survey links do not lead to previews

### ISSUE THREAD

**Issue by lexnederbragt** [carpentries/instructor-training#1078]
06-feedback.md has links to the Pre-workshop and Post-workshop surveys, and these used to be previews. Now I can't see all the questions without answering them all... Can the preview mode be re-enabeled?

**Comment by maneesha** [comment]
@fmichonneau  At some point you and I had a conversation about this but I don't remember what we talked about.  Can you help refresh my memory?

**Comment by fmichonneau** [comment]
Hi @lexnederbragt 

The new system we used for our surveys doesn't have a preview mode the same way SurveyMonkey does. 

Up until now, we had this page: https://docs.carpentries.org/topic_folders/assessment/assessment.html where the surveys are embedded in the page, but you sill have to go through the questions to see them.

I recently set up this website: https://carpentries.github.io/assessment-archives/ that lists current (and when there will be changes) past versions of our survey in a text-only format which makes it easier to browse their content. Let me know your feedback on this.
